Runtime/Core/SharedVertex.cs

```diff
@@ -63,49 +65,83 @@ public SharedVertex(SharedVertex sharedVertex)
             set { m_Vertices[i] = value; }
         }
 
-        /// <inheritdoc />
+        /// <summary>
+        /// Returns an enumerator that iterates through this collection.
+        /// </summary>
+        /// <returns>An IEnumerator object that you can use to iterate through the collection.</returns>
+        // <inheritdoc />
```

Comment:
I can remove all the lines that I hid from DocFX, if that's what you mean. I just didn't want to completely remove those lines until I was sure you were ok with that (since this tag doesn't inherit docs that are defined outside this package).

Reply:
I removed all the places where I had hidden `<inheritdoc />` from the XMLDoc comments.